docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md

```diff
@@ -33,3 +33,18 @@ OpenVINO runtime also has several execution capabilities which work on top of ot
 
 Devices similar to the ones we have used for benchmarking can be accessed using [Intel® DevCloud for the Edge](https://devcloud.intel.com/edge/), a remote development environment with access to Intel® hardware and the latest versions of the Intel® Distribution of the OpenVINO™ Toolkit. [Learn more](https://devcloud.intel.com/edge/get_started/devcloud/) or [Register here](https://inteliot.force.com/DevcloudForEdge/s/).
 
+
+## Features support matrix
+
+| Capability | CPU | GPU | GNA | VPU |
+| ---------- | --- | --- | --- | --- |
+| Hetero execution| Yes | <a href="GPU.md#hetero-execution">Yes</a> | ? | ? |
```

Comment:
@andrew-zaytsev @ilya-lavrenov  could you advice how to create a valid link to the paragraph in another file?
I've tried `[Yes](GPU.md#hetero-execution)` which works fine in markdown, but fails on docs build with the following error:
```
 unable to resolve reference to 'openvino_docs_ov_ug_supported_plugins_gpu#hetero-execution' for \ref command line (42)
```
and `<a href="GPU.md#hetero-execution">Yes</a>` which can be built, but links in rendered docs don't work.

Reply:
Yes, the links to anchors are not currently supported, please use the text form. For example;
"For more details, see the **Hetero Execution** section in the [GPU Plugin Documentation](GPU.md)